### DISCUSSION TITLE
Managing Multiple Remote Clusters with a Single FluxCD Instance

### DISCUSSION THREAD

**Discussion by JeanBrov** [fluxcd/flux2#4171] [Q&A]
Hi FluxCD community,

I'm currently exploring ways to configure a management-cluster with FluxCD installed, aiming to deploy applications across multiple remote clusters. Ideally, I'd like to avoid installing FluxCD on these remote clusters. Does anyone have an example or guide on how to achieve this?

I've come across similar inquiries in the community:

[Discussion #2258](https://github.com/fluxcd/flux2/discussions/2258)
[Discussion #3280](https://github.com/fluxcd/flux2/discussions/3280)
However, it appears that a definitive solution or working example hasn't been provided yet.

Any assistance or direction would be greatly appreciated. 

Thank you for your time and expertise!

**Comment by makkes** [comment]
I haven't updated the repo in a while but at [makkes/flux-mc-control-plane](https://github.com/makkes/flux-mc-control-plane/) you can see an example structure that manages remote clusters without necessarily installing Flux on the remotes.

**Reply by JeanBrov** [reply]
Thank you @makkes , 
I will study it.
